lab-yunjoo/gulp.js

```diff
@@ -0,0 +1,20 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+gulp.task('lint', function(){
+  return gulp.src(['*.js','lib/*.js','test/*.js'])
+  .pipe(eslint().pipe(eslint.format()))
+  .pipe(eslint.failAfterError());
+});
+
+gulp.task('mocha', function(){
+  return gulp.src(['*.js','lib/*.js','test/*.js'])
+  .pipe(mocha());
+});
+
```

Comment:
This is a good way to make sure gulp sees ALL tests, but if you have more than one test this could cause problems.  Try making Mocha's gulp.src() point to a single file - In your case testing.js

reference:
https://github.com/sindresorhus/gulp-mocha/#usage